internal/db/schema/migrations/oss/postgres/79/01_auth_oidc_prompt.up.sql

```diff
@@ -0,0 +1,105 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+-- auth_oidc_prompt_enm entries define the supported oidc auth method prompts.
+create table auth_oidc_prompt_enm (
+  name text primary key
+    constraint only_predefined_auth_oidc_prompts_allowed
+    check (
+      name in (
+        'none',
+        'login',
+        'consent',
+        'select_account'
+      )
+    )
+);
+
+insert into auth_oidc_prompt_enm (name) values
+  ('none'),
+  ('login'),
+  ('consent'),
+  ('select_account');
+
+-- auth_oidc_prompt entries are the prompts allowed for an oidc auth method.
+create table auth_oidc_prompt (
+  create_time wt_timestamp,
+  oidc_method_id wt_public_id 
+    constraint auth_oidc_method_fkey
+    references auth_oidc_method(public_id)
+    on delete cascade
+    on update cascade,
+  prompt text
+    constraint auth_oidc_prompt_enm_fkey
+    references auth_oidc_prompt_enm(name)
+    on delete restrict
+    on update cascade,
+  primary key(oidc_method_id, prompt)
+);
+comment on table auth_oidc_prompt is
+  'auth_oidc_prompt entries are the prompts allowed for an oidc auth method.  An oidc auth method may have 0 or 1 comma separated prompts value';
```

Comment:
I don't think the "comma separated" part of this comment is correct.

Aren't these the prompts "specified" for the auth method vs what's "allowed".